### DISCUSSION TITLE
goreleaser push github failed

### DISCUSSION THREAD

**Discussion by chushenmeshile** [goreleaser/goreleaser#2306] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32835062/122597487-00b1e200-d09e-11eb-9511-21bd6eecc8ba.png)
I think `GITHUB_TOKEN` is not right, but i don't know how to set it. Can give me a example? Thank you

**Comment by caarlos0** [comment, marked as answer]
seems like an intermitent issue with github... I would retry the build...

**Reply by chushenmeshile** [reply]
I can build in github workfolw, but building push GitHub locally causes this error.

**Reply by caarlos0** [reply]
nothing we can do though, the error indicate a network issue...

**Reply by chushenmeshile** [reply]
Yes, you are right, thank you
